.github/workflows/flow-semantic-release.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Semantic Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
```

Comment:
This should be using the self-hosted runner:

`runs-on: [self-hosted, Linux, medium, ephemeral]`

Reply:
Hello @rbarkerSL,
Thanks for taking the time to review this PR. I have addressed your comment.